test/e2e/networking.go

```diff
@@ -241,6 +208,50 @@ var _ = Describe("Networking", func() {
 		Expect(string(body)).To(Equal("pass"))
 	})
 
+	// Marked with [Skipped] to skip the tests by default.  These
```

Comment:
We're planning to get rid of the `[Skipped]` label (cc @ihmccreery). Where/when should these tests be run?